### PR TITLE
improve: !map webhooks 🌊

### PR DESCRIPTION
**Pull request by mistralwz**
the current one is very not pretty (it's bad because on the last commit I could not test it at all...)

I've made the embed a lot more compact but still easy to understand - clicking on the map name lets you download the map, linking more stuff, displaying extra stats. might be a bit bloated... but nicer than the current one.

might change it up a bit more after bns give more inputs

<details>
<summary>screenshot</summary>

</details>

### CODE REVIEW

**Comment by tsunyoku** [comment]
personal opinions being thrown about design before i review the code:

- the length thing is missing a clock outline imo, it just looks like a backwards L and is only obvious what it's for by it's time format.
- the combo thing can be easily misread as object count (the `x` on the end is just about a saving grace, but it looks ugly imo)
- the AR icon just has no correlation, it just seems purely random
- ^ same as this but with OD too

i know icons make it more compact but to me it is necessary that they can all be easily read and not misinterpreted, and i feel like that will just never be the case with AR, OD and combo. but the length one can be fixed for sure

i had to hover over the actual embed in your discord to understand what he AR and OD ones were, which is already a bad start because i have more context than most people and yet even i could not interpret

**Comment by mistralwz** [comment]
removed all map stats icons @ https://github.com/osuAkatsuki/bancho-service/pull/92/commits/2e53f07c0382b2d2cdf789a98848078088f52ed3 - i actually kind of liked bpm and max combo but you're right, it's not worth sacrificing readability for.

<details>
<summary>screenshot</summary>



</details>

i'm ready to get beaten up once anyone looks at the horrible code edit

File: constants/chatbotCommands.py [hunk 1]
```diff
@@ -1440,30 +1440,53 @@ async def editMap(fro: str, chan: str, message: list[str]) -> Optional[str]:
...
         colour=status_to_colour(status),
-        author=f"This {message[1]} has recieved a status update. :pencil:",
-        author_url=f'https://chimu.moe/d/{res["beatmapset_id"]}',
-        author_icon="https://cdn.discordapp.com/emojis/1160855094712078368.png",
-        image=f'https://assets.ppy.sh/beatmaps/{res["beatmapset_id"]}/covers/cover.jpg?1522396856',
+        author=f"{fro} ({token['user_id']})",
+        author_url=f"{nominator_profile_url}",
+        author_icon=f"https://a.akatsuki.gg/{token['user_id']}",
+        title=f'{mode_to_emoji(res["mode"])} {res["song_name"]}',
+        title_url=f'https://bathbot.de/osudirect/{res["beatmapset_id"]}',
```
Comment (tsunyoku) [inline constants/chatbotCommands.py]: idk about bathbot as the download... is your intention to download an osz or to redirect to ingame direct? cus we can do both without needing bathbot

Comment (mistralwz) [reply constants/chatbotCommands.py]: we're redirecting it to ingame direct! afaik you can't use `osu://` links on the embed - if that was an option i think bathbot wouldn't have made that site.

Comment (tsunyoku) [reply constants/chatbotCommands.py]: arghhhhhhhhhhh i hate more dependencies on random websites i kinda would rather it just not go to direct

Comment (mistralwz) [reply constants/chatbotCommands.py]: that's ok! let's just make that redirect to the official bancho/akatsuki listing then? i think that seems to make the most sense

Comment (tsunyoku) [reply constants/chatbotCommands.py]: yes

Comment (mistralwz) [reply constants/chatbotCommands.py]: changed things in https://github.com/osuAkatsuki/bancho-service/pull/92/commits/459d1a09985d1e6500fcb075815eef46cf7313d6 to suggested changes

Comment (mistralwz) [reply constants/chatbotCommands.py]: @tsunyoku im going to sleep but i think everything looks good now, here's the current embed...

**Review by tsunyoku — APPROVED** [review]

**Pull request merged**